src/components/Display.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import '../styles/Display.css';
+import PropTypes from 'prop-types';
+
+const Display = props => {
```

Comment:
[optional]
You can destructure it in the props in the following way.
```js
const Display = { result } => {
```
Then you can use the result directly.